packages/amplify-util-mock/src/__e2e__/model-with-maps-to.e2e.test.ts

```diff
@@ -26,11 +25,6 @@ beforeAll(async () => {
         ...defaultTransformParams.transformParameters,
         useSubUsernameForDefaultIdentityClaim: false,
       },
-      modelToDatasourceMap: new Map(
```

Comment:
Here and throughout all E2Es: the testTransform will create a dataSourceStrategies map for each model in the schema using the default DDB strategy. Note that only the test transform is wired to do that; we require actual prod code paths to specify the strategies.